File: back-end/src/main/java/com/capstone/backend/jpa/Symptom.java
package com.capstone.backend.jpa;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "symptom")
public class Symptom {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Byte id;

    @Column(name = "description", nullable = false)
    private String description;
    
    public Symptom() {}

	public Symptom(Byte id, String description) {
		super();
		this.id = id;
		this.description = description;
	}

	public Byte getId() {
		return id;
	}

	public void setId(Byte id) {
		this.id = id;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	@Override
	public String toString() {
		return "Symptom [id=" + id + ", description=" + description + "]";
	}
	
}

File: front-end/src/app/models/symptom.ts
export class Symptom {
    id: number;
    description: string;

    constructor(id?: number, description?: string) {
        this.id = id || 0;
        this.description = description || '';
    }

    get Id() {
        return this.id;
    }

    set Id(value: number) {
        this.id = value;
    }

    get Description(): string {
        return this.description;
    }

    set Description(value: string) {
        this.description = value;
    }

    toString(): string {
        return `Symptom [id=${this.id}, description=${this.description}]`;
    }
}